### ISSUE TITLE
Add dist directory to .gitignore

### ISSUE THREAD

**Issue by anshumanv** [fossasia/susi_desktop#92]
dist directory is created when we package our electron app, we don't want it to be committed hence we need to add it to .gitignore.

##### Please check the applicable fields -

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by 0x48piraj** [comment]
We should merge #93 and close this issue! :+1:

**Comment by anshumanv** [comment]
Waiting for a maintainer to be on. @0x48piraj :smile:
